src/e2e-test/java/io/cdap/plugin/bigqueryexecute/runner/TestRunnerRequired.java

```diff
@@ -25,7 +25,8 @@
 @RunWith(Cucumber.class)
 @CucumberOptions(
   features = {"src/e2e-test/features"},
-  glue = {"io.cdap.plugin.bigqueryexecute.stepsdesign", "io.cdap.plugin.common.stepsdesign", "stepsdesign"},
+  glue = {"io.cdap.plugin.bigqueryexecute.stepsdesign", "io.cdap.plugin.common.stepsdesign", "stepsdesign",
+  "io.cdap.plugin.common.stepsdesign"},
```

Comment:
Remove the change done here as well, as this is repeated.

Reply:
reverted